### ISSUE TITLE
Doesn't work on touch-screens

### ISSUE THREAD

**Issue by Magnogen** [TodePond/CellPond#86]
This'd certainly make a good app on mobile devices! I can see myself wasting _hours_ with this.

Could you add support for `touch` events too?

As for right click, it might need a long press in order to work?

**Comment by TodePond** [comment]
Yeah there's no real reason I haven't made it support touch events yet! Just been distracted by other stuff! I've been trying hard to keep all interactions done with just left-clicks so that it translates to touch well... But I've occasionally given in with some shortcuts for myself like the right-click :) 

I'm hoping to have some paddles on the right-hand-side of the screen eventually, for doing extra things, like picking a colour.
